src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[][] salaryByName = new String[names.length][2];
+        for (int i = 0; i < names.length; i++) {
+            salaryByName[i][0] = names[i];
+            salaryByName[i][1] = "0";
+        }
+        String[][] splitedData = splitData(data);
```

Comment:
The use of two-dimension arrays is discouraged as per the checklist. Try to find an alternative way to process the data without using two-dimension arrays.